roles/linux_ansible_init/files/ansible-init.py

```diff
@@ -14,6 +14,21 @@
 
 logger = logging.getLogger(__name__)
 
+SENTINEL_FILE_PATH = "/var/lib/ansible-init.done"
+
+def create_sentinel_file():
+    """
+    Creates a sentinel file to mark the service as executed.
+    """
+    try:
+        # Ensure the directory exists
+        os.makedirs(os.path.dirname(SENTINEL_FILE_PATH), exist_ok=True)
+        with open(SENTINEL_FILE_PATH, 'w') as f:
+            f.write("ansible-init executed")
```

Comment:
`pathlib` is the more modern way of doing this, e.g.:

```python
SENTINEL = pathlib.Path("/var/lib/ansible-init.done")

SENTINEL.parent.mkdir(mode = o755, parents = True, exist_ok = True)
with SENTINEL.open('w') as f:
    f.write("ansible-init succeeded")
```

Reply:
We could even use

```python
SENTINEL.touch()
```

instead of writing to the file. This is more equivalent to the previous behaviour with `ExecStartPost` 🤷‍♂️